0010-self-service/0010-self-service.md

```diff
@@ -0,0 +1,7 @@
+# Self service
+
+As a potential user I would like to create k8s clusters and/or my own control plane without having to talk to Giant Swarm.
+
+Benefits: Lower barrier to experience us
```

Comment:
What is the benefit to the user of creating k8s clusters using Giant Swarm's product?

Without talking to us, what is the "us" that they are experiencing with this?

Reply:
We might need to think about the value of the product alone here, but this should maybe also not be seen as the end state, but as a quicker entry, from where people can get in touch with us, but without having to do it before they've even tried the product. Assuming or asserting that our product alone does offer some kind of value by itself that would motivate to get in touch?